packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -34,6 +36,17 @@ function normalizeSearchInput( input = '' ) {
 	return input;
 }
 
+/**
+ * Extracts words from an input string.
+ *
+ * @param {string} input The input string.
+ *
+ * @return {Array} Words, extracted from the input string.
+ */
+function extractWords( input = '' ) {
+	return noCase( input ).split( ' ' ).filter( Boolean );
```

Comment:
Nit: I would prefer `.filter( s => !!s )`, `.filter( s => Boolean( s ) )`, or `.filter( s => s !== '' )`, as I find them much more readable, but I concede that `.filter( Boolean )` usage is pretty widespread, and has probably become an idiom at this point.

Reply:
Yeah, we're doing that in quite a bunch of places in Gutenberg already, and it's already become idiomatic in this codebase IMHO.